<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(30, 30, 30)</string>
  </property>
  <widget class="QTextBrowser" name="txtBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>501</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(212, 212, 202);</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelProgress">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>104</width>
        <height>12</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>14</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(212, 212, 202);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar
{
text-align: center;
background-color:white;
color: rgb(30,30,30);
min-height: 16px;
max-height: 16px;
}
QProgressBar::chunk 
{
background-color: rgb(140, 220, 254);
}      </string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="saveBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 30, 30);
background-color:rgb(30, 30, 30);
border: 2px solid rgb(30, 30, 30);
</string>
   </property>
   <property name="text">
    <string>Save as Exel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="exitBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 30, 30);
background-color:rgb(30, 30, 30);
border: 2px solid rgb(30, 30, 30);
</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QToolButton" name="analyzeBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 30, 30);
background-color:rgb(30, 30, 30);
border: 2px solid rgb(30, 30, 30);
</string>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
